mollie-payments-for-woocommerce/includes/mollie/wc/gateway/abstract.php

```diff
@@ -265,8 +265,8 @@ public function is_available() {
 				'sequenceType' => \Mollie\Api\Types\SequenceType::SEQUENCETYPE_ONEOFF
 			);
 
-			// Check regular payments
-			$status = $this->getAvailableMethodsInCheckout( $filters );
+			// For regular payments, check available payment methods, but ignore SSD gateway (not shown in checkout)
+			$status = ( $this->id !== 'mollie_wc_gateway_directdebit' ) ? $this->isAvailableMethodInCheckout( $filters ) : false;
```

Comment:
If DirectDebit is never available in the checkout I think it's better to move this check to the `isAvailableMethodInCheckout` method.

Reply:
I do have to check the recurring totals (during checkout) against SDD for future renewal payments.